### ISSUE TITLE
Modify IMAT GUI to use remote algorithms v2 instead of SCARFTomoReconstruction

### ISSUE THREAD

**Issue by mantid-builder** [mantidproject/mantid#12376]
This issue was originally [TRAC 11538](http://trac.mantidproject.org/mantid/ticket/11538)

Original Reporter: @FedeMPouzols 

This ticket is blocked by : 
 - [X] [TRAC11126](http://trac.mantidproject.org/mantid/ticket/11126)

There are TODO comments in the IMAT tomography GUI code that say what needs to be done: use remote algorithms with SCARF compute resource instead of the (transitional) SCARFTomoReconstruction algorithm. Just go ahead with those TODOs as soon as http://trac.mantidproject.org/mantid/ticket/11126 is merge into master. This is straightforward and just requires a bit of double-checking / testing.

Even though this is a modification in GUI code, I classify this as "Framework", as it is actually about using different algorithms and getting rid of a temporary one. The GUI functionality should remain unaffected.

The algorithm SCARFTomoReconstruction was introduced as a temporary solution, and after this it can go away. It could be removed as part of this ticket or in a follow-up ticket. In any case it would be good to kill it before 3.4.

 - - - - 
Keywords: IMAT
